frontend/src/components/header/Header.tsx

```diff
@@ -1,11 +1,16 @@
 import "./Header.css";
-import {Link} from "react-router-dom";
+import "../../assets/HomeSvg.css";
+import { Link } from "react-router-dom";
 
 function Header() {
     return (
         <header className={"header-border"}>
             <nav className={"nav-main"}>
-                <Link to={"/"}>Startseite</Link>
+                <Link to={"/"} className="svg-icon">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32" width="34" height="34">
+                        <path d="M16 2.59375L15.28125 3.28125L2.28125 16.28125L3.71875 17.71875L5 16.4375L5 28L14 28L14 18L18 18L18 28L27 28L27 16.4375L28.28125 17.71875L29.71875 16.28125L16.71875 3.28125ZM16 5.4375L25 14.4375L25 26L20 26L20 16L12 16L12 26L7 26L7 14.4375Z" fill="white"/>
+                    </svg>
```

Comment:
Tipp: In meinem Code habe ich auch SVGs verwendet. Was ich sehr praktisch fand war, daraus eine eigene Komponente zu erstellen und an den benötigten Stellen zu importieren. Das macht den Code meiner Meinung nach sauberer